Makefile

```diff
@@ -212,7 +212,7 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.2.1
-CONTROLLER_TOOLS_VERSION ?= v0.13.0
+CONTROLLER_TOOLS_VERSION ?= v0.14.0
```

Comment:
I'd suggest have a separate commit for the controller-tools version changes (Makefile, README.md, config/crd/bases/appstudio.redhat.com_dependencyupdatechecks.yaml), open a PR to main branch instead.

Reply:
see https://github.com/konflux-ci/mintmaker/pull/113